src/sidebar/util/test/annotation-metadata-test.js

```diff
@@ -488,4 +502,69 @@ describe('annotation-metadata', function () {
       assert.equal(annotationMetadata.quote(ann), null);
     });
   });
+
+  describe('hasBeenEdited', () => {
+    it('should return false if created and updated timestamps are equal', () => {
+      const annotation = fakeAnnotation({
+        created: '2020-10-28T15:35:26.822151+00:00',
+        updated: '2020-10-28T15:35:26.822151+00:00',
+      });
+      assert.isFalse(annotationMetadata.hasBeenEdited(annotation));
+    });
+
+    it('should return false if created and updated timestamps are very close', () => {
+      // Sub-second difference
+      const annotation1 = fakeAnnotation({
+        created: '2020-10-28T15:35:26.822151+00:00',
+        updated: '2020-10-28T15:35:26.822450+00:00',
+      });
+      // Slightly more than one-second difference
+      const annotation2 = fakeAnnotation({
+        created: '2020-10-28T15:35:26.822151+00:00',
+        updated: '2020-10-28T15:35:27.822450+00:00',
+      });
+      assert.isFalse(annotationMetadata.hasBeenEdited(annotation1));
+      assert.isFalse(annotationMetadata.hasBeenEdited(annotation2));
+    });
+
+    it('should return true if created and updated timestamps are meaningfully different', () => {
+      // Few seconds difference
+      const annotation1 = fakeAnnotation({
+        created: '2020-10-28T15:35:26.822151+00:00',
+        updated: '2020-10-28T15:35:29.555555+00:00',
+      });
+      // A few days
+      const annotation2 = fakeAnnotation({
+        created: '2020-10-28T15:35:26.822151+00:00',
+        updated: '2020-10-30T15:35:27.822450+00:00',
+      });
+      assert.isTrue(annotationMetadata.hasBeenEdited(annotation1));
+      assert.isTrue(annotationMetadata.hasBeenEdited(annotation2));
+    });
+
+    context('invalid created or updated dates', () => {
+      it('should return false if created and updated are equal', () => {
+        const annotation = fakeAnnotation({
+          created: 'nope',
+          updated: 'nope',
+        });
+        assert.isFalse(annotationMetadata.hasBeenEdited(annotation));
+      });
+
+      it('should return false if updated is not present', () => {
+        const annotation = fakeAnnotation({
+          created: '2020-10-28T15:35:26.822151+00:00',
+        });
+        assert.isFalse(annotationMetadata.hasBeenEdited(annotation));
+      });
+
+      it('should return false if created and updated are different but problematic', () => {
```

Comment:
```suggestion
      it('should return false if created and updated are different but cannot be parsed', () => {
```

I think it is useful to be more specific about what "problematic" means here.